.gitignore

```diff
@@ -161,3 +161,9 @@ ENV/
 # clang tooling
 compile_commands.json
 .clangd/
+
+# This name can (and probably should) be changed
+# but whatever the desired output directory for `rattler-build`, it needs to be ignored here
+# so that it doesn't get copied by `rattler` -- this leads to recursive copying
+# and eventually a `File Too Long error` which is hard to diagnose
+output/
```

Comment:
Ideally we use an output folder as we specify here:
https://github.com/rapidsai/ci-imgs/blob/ae3cc252fc2363dcebf3f245ad54808fbf9cbb06/context/condarc.tmpl#L8-L12

This is currently set to `/tmp/conda-bld-output`. We might use that, or possibly something like `/tmp/rattler-build-output`.
https://github.com/rapidsai/ci-imgs/blob/ae3cc252fc2363dcebf3f245ad54808fbf9cbb06/ci-conda.Dockerfile#L206

Reply:
yeah, that can come out, so long as the output directory is outside the source tree (which it is)